pkgs/applications/display-managers/phog/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkgconfig
+, gcr
+, glib
+, gnome-desktop
+, gtk3
+, libgudev
+, libjson
+, json-glib
+, libhandy
+, networkmanager
+, linux-pam
+, systemd
+, upower
+, wayland
+, libxkbcommon
+, python3
+, phoc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "phog";
+  version = "0.1.3";
+
+  src = fetchFromGitLab {
+    owner = "mobian1";
+    repo = "phog";
+    rev = version;
+    hash = "sha256-zny1FUYKwVXVSBGTh8AFVtMBS7dWZHTKO2gkPNPSL2M=";
+  };
+
+  # post_install is not passed two arguments, only one, so this doesn't work.
+  # This is probably a leftover from phosh/phog fork.
+  postPatch = ''
+    patchShebangs build-aux/post_install.py
+  '';
+
+  # DESTDIR needs to be set, but empty, in order to allow the post_install.py
+  # script to execute and not cause meson to crash
+  DESTDIR = "";
```

Comment:
this isn't right. on closer look, the post_install file isn't just plugin legacy from phosh: it also [compiles schemas](https://gitlab.com/mobian1/phog/-/blob/main/build-aux/post_install.py?ref_type=heads#L12). without that, phog crashes on launch trying to access gsettings stuff.

instead of setting `DESTDIR = ""` to prevent post_install from running, adding `sed -i /phog_plugins_dir/d build-aux/post_install.py` to `postPatch` allows post_install to succeed (and compile schemas) even without the plugins.